core/spec/models/spree/inventory_unit_spec.rb

```diff
@@ -142,6 +142,7 @@
     }
 
     it "should create a stock movement" do
+      expect(Spree::Deprecation).to receive(:warn)
       Spree::InventoryUnit.finalize_units!(inventory_units)
```

Comment:
why not using the new call here and remove the Deprecation `:warn` stub?

Reply:
This is a test for `Spree::InventoryUnit#finalize_units!`.